symplyphysics/laws/electricity/waveguides/power_carried_by_main_wave_of_rectangular_waveguide.py

```diff
@@ -0,0 +1,57 @@
+from sympy import (Eq, solve, sqrt,)
+from symplyphysics import (units, Quantity, Symbol, print_expression, validate_input,
+    validate_output,)
+
+
+# Description
+## A rectangular waveguide is a rectangular metal waveguide capable of supporting waves propagating along it.
```

Comment:
Don't copypaste it everywhere. Cleanup all your waveguides to only keep relevant information.